src/app/ecosystem-explorer/project-form/components/EcosystemProjectForm.tsx

```diff
@@ -0,0 +1,225 @@
+            />
+          </div>
+        </FormField>
+
+        <ControlledFormListbox<EcosystemProjectFormData>
+          name="yearJoined"
+          label="What year did your project start using Filecoin or IPFS?"
+          placeholder="Select year"
+          options={yearOptions}
+          disabled={isSubmitting}
+          innerWidth="w-40"
+        />
+
+        <ControlledFormTextarea<EcosystemProjectFormData>
+          name="briefSummary"
+          label="Brief Summary"
+          placeholder="Write a brief summary of your project..."
+          characterLimit={BRIEF_CHARACTER_LIMIT}
+          disabled={isSubmitting}
+        />
+
+        <ControlledFormTextarea<EcosystemProjectFormData>
+          name="networkUseCase"
+          label="Network Use Case"
+          placeholder="Describe how your project uses the Filecoin network..."
+          characterLimit={NETWORK_USE_CASE_CHARACTER_LIMIT}
+          disabled={isSubmitting}
+        />
+
+        <div className="flex flex-col gap-2 sm:flex-row sm:justify-evenly sm:gap-6">
+          <div className="min-w-0 sm:w-1/2">
+            <ControlledFormListbox<EcosystemProjectFormData>
+              name="category"
+              label="Category"
+              placeholder="Select category"
+              options={categoryOptions}
+              disabled={isSubmitting}
+            />
+          </div>
+          <div className="min-w-0 sm:w-1/2">
+            <ControlledFormListbox<EcosystemProjectFormData>
+              name="topic"
+              label="Topic"
+              placeholder="Select topic"
+              options={subCategoryOptions}
+              disabled={isSubmitting}
+            />
+          </div>
+        </div>
+
+        <ControlledFormFileInput<EcosystemProjectFormData>
+          name="files"
+          label="Choose a Logo for your project"
+          accept={['.png', '.jpg', '.svg', '.webp']}
+          maxSize={MAX_FILE_SIZE_IN_BYTES}
+          disabled={isSubmitting}
+          description={
+            <>
+              For best quality, please submit a white logo with a transparent
+              background, at least 1000px by 1000px, and under 100KB. You can
+              use tools like{' '}
+              <a
+                href="https://squoosh.app/"
+                className="text-brand-300 underline hover:text-brand-400"
+                rel="noopener noreferrer"
+                target="_blank"
+                aria-label='Opens "Sqoosh" in a new tab'
+              >
+                Sqoosh
+              </a>{' '}
+              to compress your image.
+            </>
+          }
+        />
+
+        <ControlledFormInput<EcosystemProjectFormData>
+          name="websiteUrl"
+          label="Website URL"
+          placeholder="Website URL"
+          type="url"
+          disabled={isSubmitting}
+        />
+
+        <ControlledFormInput<EcosystemProjectFormData, StringOrUndefined>
+          optionalLabel
```

Comment:
First time I read it it sounded like the label is optional? But actually the field is optional? Wondering if should rename the prop to optional? What do you think?

Reply:
I agree, renamed it to `addOptionalToLabel`